src/security/PeerOptions.cc

```diff
@@ -62,6 +62,13 @@ Security::PeerOptions::parse(const char *token)
         parseOptions();
     } else if (strncmp(token, "cipher=", 7) == 0) {
         sslCipher = SBuf(token + 7);
+#if USE_OPENSSL
+        // everything okay
+#elif USE_GNUTLS
+        debugs(3, DBG_PARSE_NOTE(1), "WARNING: SSL cipher= not supported. Uses GnuTLS tls-options= parameters instead.");
+#else
+        debugs(3, DBG_PARSE_NOTE(1), "WARNING: SSL cipher= option requires --with-openssl.");
```

Comment:
Please remove `SSL `.

Reply:
done. and for other mentions in relation to ciphers.